lib/scout_apm/agent.rb

```diff
@@ -69,6 +68,8 @@ def start(opts={})
 
       install unless context.installed?
 
+      instrument_manager.install! # noop if it was installed earlier
```

Comment:
Does this need an `if should_load_instruments?` check?

Reply:
no, since all that would do is check `monitor`, which we did at the top of this method anyway.

Reply:
What if `should_install_instruments?` changes? Leave that decision up to `should_install_instruments?` instead of assuming it will be the same as the `monitor` check on L62. That also makes it consistent with how it's used on L40

Reply:
Added in 3950108 after I merged